chart/templates/environment-secrets.yaml

```diff
@@ -44,3 +44,39 @@ spec:
         - regexp:
             source: "[^a-zA-Z0-9 -]"
             target: "_"
+
+---
+apiVersion: external-secrets.io/v1beta1
```

Comment:
this section may not be needed - since it's using the same credential as the pre-existing SecretStore, as long as the role is added to that account we should not have an issue using the existing SecretStore

Reply:
It is needed, `spec.provider.aws.service` is changing to `SecretsManager` and not `ParameterStore`